L/libcvrpsep/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libcvrpsep"
+version = v"0.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/chkwon/CVRPSEP.git", "c4d1720ffafcc174e39f65c7a90146cb82b68e44")
```

Comment:
Why using a fork and not the upstream package?

Reply:
The [original](https://econ.au.dk/research/researcher-websites/jens-lysgaard/cvrpsep/) didn't use consistent capitalizations in filenames, which made the compilation not work properly. Also some parts of the code didn't compile well. Those issues were fixed and autoconf was used in the fork.